buildSrc/build.gradle.kts

```diff
@@ -4,4 +4,9 @@ plugins {
 
 repositories {
     jcenter()
+    google()
```

Comment:
I think I remember reading somewhere that it's a good idea to have Google listed first so it's less likely that any of those critical libraries might be spoofed by other versions.

Reply:
@f2rf2r Can you address this comment?